### ISSUE TITLE
event report including images as html

### ISSUE THREAD

**Issue by salikatt** [danielskaare/isurvey_tools#16]
I have been thinking about events and images as en export from NM.
It's possible to export the files including path to images and we should try to use that.

I think the best way to create a report of the events and corresponding image is using html, where we're using a relative link to a sub-folder with all the images.
I believe there is relative good functionality within python to write this html that could be pretty simple as a start. We can consider CSS later.

I also know that there is some functionality inside QGIS with regards to image-files, but i have not found the time to explore the function thoroughly.

Whats your thought Daniel?

**Comment by danielskaare** [comment]
Sounds like a good way to go. Have you considered to let the dataflow go through the masterfile? With the relative link in a sepperate column? Then we add the path up to the relative link in QGIS.

HTML sounds like a good solution and then add a print to pdf from there. If you have any examples or projects please send link, email or upload files here  👍

**Comment by salikatt** [comment]
yes, good idea, could have PGS to check the route via mastefile.
I believe PK have some short surveys with a few evens that could be used for testing.
